File: gita80/if/task1/main.dart
import 'dart:io';
import 'dart:math';

void main() {
  stdout.write("a = ");
  String aStr = stdin.readLineSync()??'';
  stdout.write("b = ");
  String bStr = stdin.readLineSync()??'';
  stdout.write("c = ");
  String cStr = stdin.readLineSync()??'';

  int a = int.tryParse(aStr)??0;
  int b = int.tryParse(bStr)??0;
  int c = int.tryParse(cStr)??0;

  int minimumV1 = a;
  if(minimumV1>b)minimumV1=b;
  if(minimumV1>c)minimumV1=c;
  print("Minimum V1: ${minimumV1}");

  int minimumV2 = min(a, min(b,c));
  print("Minimum V2: ${minimumV2}");
}

File: gita80/if/task1/main.cpp
#include <iostream>
#include <cmath>
using namespace std;


int main() {
    int a,b,c;
    cout<<"a = ";
    cin>>a;
    cout<<"b = ";
    cin>>b;
    cout<<"c = ";
    cin>>c;
    int minimumV1 = a;
    if(minimumV1>b) minimumV1 = b;
    if(minimumV1>c) minimumV1 = c;
    cout<<"Min V1: "<<minimumV1<<endl;

    int minimumV2 = min(a,min(b,c));
    cout<<"Min V2: "<<minimumV2<<endl;

    return 0;
}
